src/relay/op/nn/convolution.cc

```diff
@@ -244,7 +250,12 @@ bool Conv2DRel(const Array<Type>& types, int num_inputs, const Attrs& attrs,
     ICHECK_EQ(param->dilation.size(), 2);
     Array<IndexExpr> wshape;
 
-    if (is_depthwise) {
+    if (is_group) {
+      // infer weight's shape for group convolution
+      wshape = {{param->groups, indexdiv(param->channels, param->groups),
+                 indexdiv(dshape_nchw[1], param->groups), param->kernel_size[0],
+                 param->kernel_size[1]}};
```

Comment:
I'm pretty sure we already support group convolution and this one looks unusual. If this is DNNL specific, can you come up with a better name than `is_group`?

Reply:
DNNL prefer to use `GOIHWxg` layout for group conv's weights. It seems that the layout cannot be transformed from `OIHW` to `GOIHWxg` directly. Because the dims are not matched for this two layout. Have this kind of transformation already been  supported? If not, how about use `is_dnnl_group_conv` instead of `is_group`?

Reply:
I don't think such transform is possible now. Yeah, `is_dnnl_group_conv` sounds better.

Reply:
OK, I will change the name :)